examples/research_projects/seq2seq-distillation/utils copy.py

```diff
@@ -1,645 +0,0 @@
-import itertools
```

Comment:
this file was just a copy of `examples/research_projects/seq2seq-distillation/utils.py`